src/DeferredFutures.jl

```diff
@@ -58,6 +89,13 @@ function finalize_ref(ref::DeferredRemoteRef)
     return nothing
 end
 
```

Comment:
Given that you're exporting `DeferredRemoteRef` for folks to subtype, I'd add a docstring to `finalize_ref` to explain what's going on when `finalize` is called on a `DeferredRemoteRef`.